cleanup-oracle.sh

```diff
@@ -276,14 +259,48 @@ else
     echo "Found Ansible: $(type ansible-playbook)"
 fi
 
+# Initialize command array
+declare -a CMD_ARRAY=()
+
+CMD_ARRAY+=($ANSIBLE_PLAYBOOK)
+CMD_ARRAY+=(-i "$INVENTORY_FILE")
+
+if [[ -n "$ANSIBLE_PARAMS" ]]; then
+  echo "Processing ANSIBLE_PARAMS string: [$ANSIBLE_PARAMS]"
+  CMD_ARRAY+=(-e "$ANSIBLE_PARAMS")
+fi
+
+# Add any passthrough arguments from the script command line
+if [[ "$#" -gt 0 ]]; then
+  CMD_ARRAY+=("$@")
+fi
+
+# Add the JSON arguments. Each part must be a separate array element
+if [[ -n "${ORA_ASM_DISKS_JSON}" ]]; then
+  # The '-e' is one element. The JSON string is the next.
+  CMD_ARRAY+=(-e "{\"asm_disk_input\":${ORA_ASM_DISKS_JSON}}")
```

Comment:
Nit: I try to avoid explicit escapes where possible.  So maybe something like
CMD_ARRAY+=(-e '{"asm_disk_input":